tests/e2e/e2e_suite_test.go

```diff
@@ -15,11 +15,18 @@ var ctx context.Context
 var token string
 var clusterID string
 
+func GetToken() string {
```

Comment:
- Why it is renamed to _test?

Reply:
because it is a test suite - it replaces the init_test file